generated/user_existence/user_existence.json

```diff
@@ -0,0 +1,29 @@
+{
+  "primary_applicant": {
+    "id": "58bf3bb6e41b0738bf0000ce",
+    "user_id": "58bf3b87e41b0738bf0000cc",
+    "first_name": "Loren",
+    "last_name": "Dimallo"
+  },
+  "employers": [
+    {
+      "employer": {
+        "id": "58bf321ee41b0738bf00000c",
+        "legal_name": "BriCol",
+        "phone": "(783) 786-7857"
```

Comment:
where are we getting this phone number? is it the first of the staff members? Or the primary office location phone number? I think we might want to just duplicate the same contact info structure the broker sees for their employers: 
https://github.com/dchealthlink/HBX-mobile-app-APIs/blob/master/generated/broker_1/broker_details.json#L81